maven-plugin-api/src/main/mdo/plugin.mdo

```diff
@@ -91,6 +91,12 @@ under the License.
           <type>boolean</type>
           <defaultValue>true</defaultValue>
         </field>
+        <field>
+          <name>requiredJavaVersion</name>
+          <version>1.1.0</version>
+          <description>A version range which specifies the supported Java versions.</description>
```

Comment:
This supports the same syntax as JDK profile activation (https://maven.apache.org/guides/introduction/introduction-to-profiles.html#details-on-profile-activation)